packages/db/src/models/granule.ts

```diff
@@ -164,8 +164,7 @@ export default class GranulePgModel extends BasePgModel<PostgresGranule, Postgre
         // exist at all
         upsertQuery.whereNotExists(exclusionClause);
       }
-      upsertQuery.returning('*');
-      return await upsertQuery;
+      return await upsertQuery.returning('*');
```

Comment:
Fix typescript compilation error, since upsertQuery is assumed to have type number[], upsertQuery.returning('*') is assumed to have type PostgresGranuleRecord[]